include/ignition/gui/Application.hh

```diff
@@ -96,11 +96,11 @@ namespace ignition
       /// and plugins. This function doesn't instantiate the plugins, it just
       /// keeps them in memory and they can be applied later by either
       /// instantiating a window or several dialogs.
-      /// \param[in] _config Full path to configuration file.
+      /// \param[in] _pluginElem Full path to configuration file.
```

Comment:
nit: Renaming to `_path` would be more clear or `_pluginPath` but this is used to load configs too. For consistency, we should also update the variable in [Application.cc](https://github.com/ignitionrobotics/ign-gui/blob/d9c78efce068db6c85cfa37d3fcadfae9ac29a24/src/Application.cc#L209)
```suggestion
      /// \param[in] _path Full path to configuration file.
```